### ISSUE TITLE
Safari: timer restart button is darker in toolbox

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/wave-interference#514]
Device
MacBook Air and iPad 6th Gen
OS
MacOS 10.15 and iPadOS 14.3
Browser
Safari
Problem Description
For https://github.com/phetsims/QA/issues/580. Does not occur in published.
The restart timer button on the timer is disabled while it is in the toolbox since the timer is at 0. However, in Safari, what that looks like in the box and out of the box is different. In the box, the button is darker. Out of the box, and in other browsers, the button is lighter.
Visuals:
Win 10 Chrome:
<img width="187" alt="buttoncolorwinchrome" src="https://user-images.githubusercontent.com/41024075/104514414-745f3880-55ae-11eb-94af-5e9a8e03bf84.png">
On iPad:
![Image from iOS (1)](https://user-images.githubusercontent.com/41024075/104514416-74f7cf00-55ae-11eb-9c4e-0aa17cd58890.png)
![Image from iOS](https://user-images.githubusercontent.com/41024075/104514419-75906580-55ae-11eb-9b0b-ead086d15558.png)

**Comment by samreid** [comment]
I wonder if a filter is getting applied twice?

**Comment by samreid** [comment]
@jonathanolson does this seem like it is related to the scenery filters?

**Comment by samreid** [comment]
I confirmed this problem is happening on Mac Safari but not Mac Chrome on https://phet-dev.colorado.edu/html/waves-intro/1.1.0-rc.3/phet/waves-intro_all_phet.html.  It is also happening on master.

**Comment by jonathanolson** [comment]
Yes, seems like https://github.com/phetsims/scenery/issues/1142. Is this blocking for this sim (presumably?)

**Comment by arouinfar** [comment]
In https://github.com/phetsims/scenery/issues/1142#issuecomment-764834443 we decided this is not a blocking issue.

**Comment by samreid** [comment]
On hold until https://github.com/phetsims/scenery/issues/1142 has progress.

**Comment by samreid** [comment]
This problem may have been solved by safari. Here are: mac/chrome + mac/safari + mac/safari screenshot:

<img width="775" height="502" alt="Image" src="https://github.com/user-attachments/assets/9ac18f0b-e325-46bd-9ed9-560d2ff6359d" />

I'll mark this for testing in next QA test.
